internal/cmd/beta/beta.go

```diff
@@ -0,0 +1,35 @@
+package beta
+
+import (
+	"github.com/stackitcloud/stackit-cli/internal/cmd/beta/sqlserverflex"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/args"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/examples"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/print"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/utils"
+
+	"github.com/spf13/cobra"
+)
+
+func NewCmd(p *print.Printer) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "beta",
+		Short: "Contains Beta STACKIT CLI commands",
+		Long:  "Contains Beta STACKIT CLI commands.",
```

Comment:
Do you think it's worth adding a bit more details like "The commands under this group are still BETA and functionality may be incomplete or have breaking changes" or not?